pkg/reconciler/ingress/resources/gateway.go

```diff
@@ -41,7 +42,11 @@ import (
 )
 
 // GatewayHTTPPort is the HTTP port the gateways listen on.
-const GatewayHTTPPort = 80
+const (
+	GatewayHTTPPort       = 80
+	DNS1123LabelMaxLength = 63 // Public for testing only.
```

Comment:
```suggestion
	dns1123LabelMaxLength = 63
```

I think this is not used for testing :thinking:

Reply:
Done